build-parent/pom.xml

```diff
@@ -71,10 +71,11 @@
         <version.jakarta.cdi>4.0.1</version.jakarta.cdi>
         <version.jakarta.inject>2.0.1</version.jakarta.inject>
         <version.jakarta.annotation>3.0.0</version.jakarta.annotation>
+        <version.jakarta.wr.rs>3.1.0</version.jakarta.wr.rs>
         <version.javax.servlet>3.1.0</version.javax.servlet>
 
 <!--        Spark Default Dependencies.  See `spark-*` profiles below for alternative sets -->
-        <version.spark>3.4.0</version.spark>
+        <version.spark>3.5.2</version.spark>
```

Comment:
Refs #55 

We might go ahead an implement the migrations identified in that ticket as part of the 1.10 migrations.